lib/buy/moonpay/moonpay_provider.dart

```diff
@@ -329,13 +330,16 @@ class MoonPayProvider extends BuyProvider {
 
     return Order(
         id: id,
-        provider: BuyProviderDescription.moonPay,
+        source: OrderSourceDescription.buy,
+        buyProvider: BuyProviderDescription.moonPay,
         transferId: id,
         state: state,
         createdAt: createdAt,
         amount: amount.toString(),
         receiveAddress: wallet.walletAddresses.address,
-        walletId: wallet.id);
+        walletId: wallet.id,
+        receiveAmount: '',
+        quantity: '1');
```

Comment:
would prefer if they are not required then, instead of passing dummy data

Reply:
removed